docs/containers/kubernetes/kubernetes-operator.rst

```diff
@@ -7,6 +7,9 @@ Run CrateDB with Kubernetes Operator
 The `CrateDB Kubernetes Operator`_ provides a convenient way to run `CrateDB`_
 clusters inside Kubernetes.
 
+Using the operator we can just deploy a `CrateDB` resource without having to
```

Comment:
In one of the other `.. NOTE::` boxes, CrateDB is wrapped in double quotes when speaking about it as a custom resource (in Sphinx, single backtics are for italic, double backticks for inline code).